static/app/components/events/interfaces/crashContent/stackTrace/content.spec.tsx

```diff
@@ -155,6 +155,68 @@ describe('StackTrace', function () {
     expect(omittedFrames).toBeInTheDocument();
   });
 
+  describe('with stacktrace improvements feature flag enabled', function () {
+    const organization = TestStubs.Organization({
+      features: ['issue-details-stacktrace-improvements'],
+    });
+
+    it('does not render non in app tags', function () {
+      const dataFrames = [...data.frames];
+      dataFrames[0] = {...dataFrames[0], inApp: false};
+
+      const newData = {
+        ...data,
+        frames: dataFrames,
+      };
+
+      renderedComponent({
+        organization,
+        data: newData,
+      });
+
+      expect(screen.queryByText('System')).not.toBeInTheDocument();
+    });
+
+    it('displays a toggle button when there is more than one non-inapp frame', function () {
+      const dataFrames = [...data.frames];
+      dataFrames[0] = {...dataFrames[0], inApp: true};
+
+      const newData = {
+        ...data,
+        frames: dataFrames,
+      };
+
+      renderedComponent({
+        organization,
+        data: newData,
+        includeSystemFrames: false,
+      });
+
+      expect(screen.getByText('Show 3 more frames')).toBeInTheDocument();
+    });
+
+    it('does not display a toggle button when there is only one non-inapp frame', function () {
+      const dataFrames = [...data.frames];
+      dataFrames[0] = {...dataFrames[0], inApp: true};
+      dataFrames[2] = {...dataFrames[2], inApp: true};
+      dataFrames[4] = {...dataFrames[4], inApp: true};
+
+      const newData = {
+        ...data,
+        frames: dataFrames,
+      };
+
+      renderedComponent({
+        organization,
+        data: newData,
+        includeSystemFrames: false,
+      });
+
+      expect(screen.queryByText('more frames')).not.toBeInTheDocument();
```

Comment:
This will never be true even if something does exist with that text. If you use regex though, it will do what you want it to:

https://testing-library.com/docs/dom-testing-library/cheatsheet/#text-match-options